src/Pim/index.tsx

```diff
@@ -395,25 +398,30 @@ class Pim extends React.PureComponent {
     const { collectionsAddressBook, collectionsCalendar, collectionsTaskList, addressBookItems, calendarItems, taskListItems } = itemsSelector(this.props);
     const [contacts, events, tasks] = itemValuesSelector(this.props);
 
-    const birthdays = contacts.filter((c) => c.bday).map((c) => {
-      let { year, month, day } = c.bday._time;
-      year = year || '2020';
-      if (month < 10) {
-        month = '0' + month;
-      }
-      if (day < 10) {
-        day = '0' + day;
-      }
-
-      const event = 
-        'BEGIN:VEVENT\n' +
-        `SUMMARY:${c.fn}'s Birthday\n` +
-        `DTSTART;VALUE=DATE:${year}${month}${day}\n` +
-        'RRULE:FREQ=YEARLY\n' +
-        'END:VEVENT';
-      return new EventType(parseString(event));
-    });
-
+    let birthdays: EventType[] = [];
+
+    if (store.getState().settings.showBirthdayCalendar) {
```

Comment:
This is not the correct way to do it, because it won't update when the store updates.
It's actually very annoying to use Redux without react hooks though, so maybe if you can't easily fix it to get it from the store, let's leave it like this and add a "FIXME" comment, so when we change it to hooks (I can do it after this PR is merged) it will be fixed.